autocpenv.py

```diff
@@ -156,14 +161,14 @@ def resolve_from_job_scenefile(self, job):
         except cpenv.ResolveError:
             pass
 
-    def resolve_from_job_plugin(self, plugin_mapping, job_plugin):
+    def resolve_from_job_plugin(self, plugin_setting, job_plugin):
```

Comment:
Can you leave this variable named `plugin_mapping`? It's a little unclear to me that it's called `plugin_setting` now.

Maybe you could add an additional argument here that is actually a string called `plugin_setting`?
For example:
```
def resolve_from_job_plugin(self, plugin_setting, plugin_mapping, job_plugin):
    ...
```
When calling it the plugin_setting could be the name of the actual setting, that would have the benefit of clarifying the logging message as well.
```
self.resolve_from_job_plugin('Forced Plugin Mapping', self.GetConfigEntry('forced_plugin_mapping'), job_plugin)
```

Reply:
This has been changed back in the latest push